pragma-node/src/infra/repositories/onchain_repository.rs

```diff
@@ -1,16 +1,36 @@
 use bigdecimal::BigDecimal;
 
 use deadpool_diesel::postgres::Pool;
-use diesel::sql_types::{BigInt, Numeric, Text, Timestamptz};
+use diesel::sql_types::{BigInt, Numeric, Text, Timestamp, VarChar};
 use diesel::{Queryable, QueryableByName, RunQueryDsl};
 
 use pragma_entities::error::{adapt_infra_error, InfraError};
 use pragma_monitoring::models::SpotEntry;
 
-use crate::handlers::entries::{AggregationMode, Network, OnchainEntry};
+use crate::handlers::entries::{AggregationMode, Checkpoint, Network, OnchainEntry};
+use crate::utils::format_bigdecimal_price;
 
 const BACKWARD_TIMESTAMP_INTERVAL: &str = "1 hour";
 
+#[allow(dead_code)]
```

Comment:
Why this ?
Also we should probably create an enum DataType rather. All of this will be included in the rust sdk in the future

Reply:
Also we should probably move Pragma common thing to the pragma-common crate for now

Reply:
cause `FutureEntry` is never used yet (we only deal with spot for now):
```rust
error: variant `FutureEntry` is never constructed
  --> pragma-node/src/infra/repositories/onchain_repository.rs:18:5
   |
15 | enum TableType {
   |      --------- variant in this enum
...
18 |     FutureEntry,
   |     ^^^^^^^^^^^
   |
   = note: `-D dead-code` implied by `-D warnings`
```

Updated the name of the enum to `DataType` 🫡